pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -33,13 +44,15 @@ let
     libxkbcommon
     dbus
     libglvnd
-    zlib
     egl-wayland
     wayland
     libGLU
     libGL
-    fontconfig
-    freetype
+  ];
+  darwinRuntimeDeps = [
```

Comment:
Please move this directly into buildInputs behind a lib.optionals.

Reply:
Should I move the Linux dependencies as well?

Reply:
Yes please.